src/cartesian/Scatter.tsx

```diff
@@ -207,8 +240,57 @@ function ScatterLine({ points, props }: { points: ReadonlyArray<ScatterPointItem
   );
 }
 
+function ScatterLabelListProvider({
```

Comment:
I'm not sure I like that labels are now below points for scatter. Feel like words should always render on top even if both are bad. People like to render large `z` scatter items and then put text in those

Also why is it below for scatter but not for other charts?

Reply:
Because they are rendered together with error bars and error bars are better looking under points.

Proper fix would be portals, then we could have error bars below and labels on top.

Different default for label positions would help too.

Reply:
Ah yeah. Another thing I suppose